apps/website/src/components/courses/GroupSwitchModal.tsx

```diff
@@ -461,7 +462,7 @@ const GroupSwitchModal: React.FC<GroupSwitchModalProps> = ({
                     className="mx-auto"
                     target="_blank"
                     rel="noopener noreferrer"
-                    url={`https://availability.bluedot.org/form/bluedot-course?email=${encodeURIComponent(auth.email)}&utm_source=bluedot-group-switch-modal`}
+                    url={`https://availability.bluedot.org/form/bluedot-course?email=${encodeURIComponent(user.email)}&utm_source=bluedot-group-switch-modal`}
```

Comment:
This was the one place where the localStorage email was used directly rather than getting it from the user returned from the api